CMakeLists.txt

```
@@ -2522,7 +2522,7 @@ elseif(PSP)
     )
 
 elseif(PS2)
-  list(APPEND EXTRA_CFLAGS "-DPS2")
+  list(APPEND EXTRA_CFLAGS "-DPS2 -D__PS2__")
```

This is a list, so you need to do
```suggestion
  list(APPEND EXTRA_CFLAGS -DPS2 -D__PS2__)
```